### DISCUSSION TITLE
Nowy opis listy

### DISCUSSION THREAD

**Discussion by ClusterByte-x** [FiltersHeroes/KAD#1706] [Pomysły]
Zmiana z:

> Filtry chroniące przed różnego rodzaju przekrętami i oszustwami, takimi jak np. subskrypcje SMS, wirusy i fałszywe sklepy internetowe. Oprócz własnego zestawu, zawierają one również wpisy z list stworzonych przez takie zespoły jak np. CERT Polska, legalniewsieci.pl i KNF.

Na, propozycja nr 1:

> Filtry chroniące przed internetowymi oszustwami w polskiej sieci - phishingiem, subskrypcjami SMS, fałszywymi sklepami i złośliwymi stronami. Lista rozbudowana jest dodatkowo o wpisy: CERT Polska, CERT Orange Polska, KNF, legalniewsieci.pl i CyberRescue.

Krótsza, zwięźlejsza forma i zaznaczenie dotyczące wyłącznie polskojęzycznej zawartości filtra.

**Comment by hawkeye116477** [comment]
Propozycja nr 2

> Filtry chroniące przed różnymi zagrożeniami w polskiej sieci, takimi jak wirusy, fałszywe sklepy i subskrypcje SMS. Dodatkowo rozbudowane są one o wpisy stworzone przez takie zespoły jak CERT Polska, legalniewsieci.pl i KNF.

Propozycja nr 3

> Filtry chroniące przed różnymi zagrożeniami w polskiej sieci, takimi jak wirusy, fałszywe sklepy i subskrypcje SMS. Dodatkowo rozbudowane są one o wpisy stworzone przez CERT Polska, CERT Orange Polska, legalniewsieci.pl, KNF i CyberRescue.

**Comment by AroKol78** [comment]
nie będę oryginalny
(`chroniące` > `pomagające chronić` nic nie chroni w 100%)
Propozycja nr 4


> Filtry pomagające chronić przed różnymi zagrożeniami w polskiej sieci, takimi jak oszustwa, wirusy, fałszywe sklepy i subskrypcje SMS.
> Dodatkowo lista zawiera wpisy stworzone przez CERT Polska, CERT Orange Polska, legalniewsieci.pl, KNF i CyberRescue.

**Reply by hawkeye116477** [reply]
Raczej oczywiste.

**Reply by ClusterByte-x** [reply]
Skoro już się czepiamy... 😛Wydaje mi się, że wpisanie tam „wirusy” jest niewłaściwe. Bardziej zainfekowane/złośliwe strony, gdyż sama lista przecież nie rozpoznaje takich zagrożeń jak wirusy (chociaż rozumiem użycie potoczne).

**Comment by hawkeye116477** [comment]
Ostateczny opis:

> Filtry chroniące przed różnymi zagrożeniami w polskiej sieci, takimi jak wirusy, fałszywe sklepy i subskrypcje SMS. Dodatkowo zawierają one wpisy stworzone przez CERT Polska, legalniewsieci.pl, KNF i inne zespoły.
